### PR TITLE
Add new bid adapter for ConnectAd

### PR DESCRIPTION
**Pull request by rtuschkany**
## Type of change
- [X] New bidder adapter

## Description of change
New Adapter for ConnectAd (formerly ServerBid Alias)

```
{
  bidder: 'connectad',
  params: {
         siteId: 123456,
         networkId: 123456
  }
}
```

- contact email of the adapter’s maintainer: rtuschkany@connectad.io
- [X] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/
https://github.com/prebid/prebid.github.io/pull/1785

## Other information
ConnectAd is currently a bidder-alias of the ServerBid Adapter. As the ServerBid-Adapter is not maintained anymore, please add our new ConnectAd Adapter.

### CODE REVIEW

**Comment by rtuschkany** [comment]
Hi! Any update on our PR? Thanks!

**Comment by rtuschkany** [comment]
@idettman @mike-chowla Hi, can someone pls give us an update on that PR. Thanks!

File: modules/connectadBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,299 @@
+import * as utils from '../src/utils';
```
Comment (idettman) [inline modules/connectadBidAdapter.js]: all imports must use the module plus the extension, ie: '`../src/utils.js'`

File: test/spec/modules/connectadBidAdapter_spec.js [hunk 2]
```diff
@@ -0,0 +1,454 @@
+import {expect} from 'chai';
```
Comment (idettman) [inline test/spec/modules/connectadBidAdapter_spec.js]: update imports to use module plus extension in test file as well.

**Comment by rtuschkany** [comment]
@idettman @mike-chowla thanks for reviewing! I've fixed all imports.

**Comment by rtuschkany** [comment]
@mike-chowla hi mike! just a quick reminder as I've received a stale message. Thx!

**Comment by rtuschkany** [comment]
@mike-chowla @idettman hey guys! Any update on this? We are already waiting for two months to get this approved. Thx!

**Pull request merged**